### PR TITLE
Update amcrest docs to use config flow

### PR DESCRIPTION
**Pull request by flacjacket**
## Proposed change

With the corresponding PR in the core repo, update the Amcrest documentation to show the config flow setup.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: home-assistant/core#80453
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/amcrest.markdown [hunk 1]
```diff
@@ -17,6 +17,7 @@ ha_platforms:
 ha_codeowners:
   - '@flacjacket'
 ha_integration_type: integration
+ha_config_flow: true
 ---
 
 The `amcrest` camera platform allows you to integrate your [Amcrest](https://amcrest.com/) or Dahua IP camera or doorbell in Home Assistant.
```
Comment (frenck) [inline source/_integrations/amcrest.markdown]: It became an integration now

```suggestion
The Amcrest integration allows you to integrate your [Amcrest](https://amcrest.com/) or Dahua IP camera or doorbell in Home Assistant.
```

File: source/_integrations/amcrest.markdown [hunk 2]
```diff
@@ -167,7 +40,9 @@ Newer Amcrest firmware may not work, then `rtsp` is recommended instead.
 make sure to follow the steps mentioned at [FFmpeg](/integrations/ffmpeg/)
```
Comment (frenck) [inline source/_integrations/amcrest.markdown]: Does this still apply even? It can be a dependency, right?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/amcrest.markdown (hunk 2 above)
Comment (flacjacket) [reply source/_integrations/amcrest.markdown]: I tried running with just the default config and an Amcrest camera with rtsp, and it looks like everything is able to run just fine with that, so I can remove this.

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Pull request closed without merging**